code/game/objects/structures/trash_pile_vr.dm

```diff
@@ -114,166 +116,98 @@
+					prob(4);/obj/item/device/pda,
+					prob(4);/obj/item/device/radio/headset,
+					prob(2);/obj/item/device/paicard,
+					prob(3);/obj/item/weapon/storage/box/sinpockets,
+					prob(2);/obj/item/weapon/storage/pill_bottle/paracetamol,
+					prob(1);/obj/item/pizzavoucher,
+					prob(2);/obj/item/device/nif/bad,
+					prob(2);/obj/item/weapon/card/emag_broken,
+					prob(1);/obj/item/device/survivalcapsule/popcabin)
 
 	var/obj/item/I = new path()
 	return I
 
 /obj/structure/trash_pile/proc/produce_beta_item()
-	var/path = pick(prob(6);/obj/item/weapon/storage/pill_bottle/paracetamol,
-					prob(4);/obj/item/weapon/storage/pill_bottle/happy,
-					prob(4);/obj/item/weapon/storage/pill_bottle/zoom,
-					prob(4);/obj/item/seeds/ambrosiavulgarisseed,
+	var/path = pick(prob(2);/obj/item/clothing/under/fluff/latexmaid,
+					prob(2);/obj/item/clothing/under/hyperfiber,
 					prob(4);/obj/item/weapon/gun/energy/sizegun,
-					prob(3);/obj/item/weapon/material/butterfly,
-					prob(3);/obj/item/weapon/material/butterfly/switchblade,
-					prob(3);/obj/item/clothing/gloves/knuckledusters,
-					prob(3);/obj/item/weapon/reagent_containers/syringe/drugs,
-					prob(2);/obj/item/weapon/implanter/sizecontrol,
+					prob(2);/obj/item/clothing/under/hyperfiber/bluespace,
+					prob(1);/obj/item/weapon/implanter/sizecontrol,
 					prob(2);/obj/item/weapon/handcuffs/fuzzy,
 					prob(2);/obj/item/weapon/handcuffs/legcuffs/fuzzy,
-					prob(2);/obj/item/weapon/storage/box/syndie_kit/spy,
-					prob(2);/obj/item/weapon/grenade/anti_photon,
-					prob(2);/obj/item/clothing/under/hyperfiber/bluespace,
-					prob(2);/obj/item/weapon/reagent_containers/glass/beaker/vial/amorphorovir,
-					prob(1);/obj/item/clothing/suit/storage/vest/heavy/merc,
-					prob(1);/obj/item/device/nif/bad,
-					prob(1);/obj/item/device/radio_jammer,
-					prob(1);/obj/item/device/sleevemate,
+					prob(1);/obj/item/weapon/reagent_containers/glass/beaker/vial/amorphorovir,
+					prob(2);/obj/item/device/sleevemate,
 					prob(1);/obj/item/device/bodysnatcher,
-					prob(1);/obj/item/weapon/beartrap,
-					prob(1);/obj/item/weapon/cell/hyper/empty,
-					prob(1);/obj/item/weapon/disk/nifsoft/compliance,
-					prob(1);/obj/item/weapon/material/knife/tacknife,
-					prob(1);/obj/item/weapon/storage/box/survival/space,
-					prob(1);/obj/item/weapon/storage/secure/briefcase/trashmoney,
-					prob(1);/obj/item/device/survivalcapsule/popcabin,
-					prob(1);/obj/item/weapon/reagent_containers/syringe/steroid,
 					prob(1);/obj/item/capture_crystal)
 
 	var/obj/item/I = new path()
 	return I
 
 /obj/structure/trash_pile/proc/produce_gamma_item()
-	var/path = pick_n_take(unique_gamma)
+	var/path = pick(prob(4);/obj/item/weapon/material/butterfly,
+					prob(4);/obj/item/weapon/material/butterfly/switchblade,
+					prob(4);/obj/item/clothing/gloves/knuckledusters,
+					prob(2);/obj/item/weapon/material/knife/tacknife,
+					prob(2);/obj/item/clothing/shoes/syndigaloshes,
```

Comment:
This can probably be bumped up to Alpha, since if I recall correctly these are just brown shoes which are anti-slip.